tools-v2/pkg/cli/command/curvebs/stop/volumeSnapshot/stop_volume.go

```diff
@@ -0,0 +1,126 @@
+package volumeSnapshot
+
+import (
+	"encoding/json"
+	"strconv"
+	"time"
+
+	cmderror "github.com/opencurve/curve/tools-v2/internal/error"
+	cobrautil "github.com/opencurve/curve/tools-v2/internal/utils"
+	basecmd "github.com/opencurve/curve/tools-v2/pkg/cli/command"
+)
+
+const (
+	VERSION = "0.0.6"
+	LIMIT   = "100"
+	OFFSET  = "0"
+)
```

Comment:
The variable place global floder is better? because all snapshot command wiil use it like `NewSubUri` function.